pkg/imagestore/imagestore.go

```diff
@@ -268,17 +273,45 @@ func isoFileName(imageType, openshiftVersion, version, arch string) string {
 	return fmt.Sprintf("rhcos-%s-%s-%s-%s.iso", imageType, openshiftVersion, version, arch)
 }
 
-func buildRootfsURL(scheme, host, arch, version string) string {
+func buildRootfsURL(scheme, host, arch, version string) (string, error) {
+	if strings.Contains(host, ":") && !strings.Contains(host, "http") {
```

Comment:
Should probably check for host starting with http rather than contains.

Reply:
Right, changed to HasPrefix